drivers/virtualbox/vbm.go

```diff
@@ -15,8 +15,6 @@ import (
 )
 
 var (
-	reVMNameUUID      = regexp.MustCompile(`"(.+)" {([0-9a-f-]+)}`)
```

Comment:
At any rate, maybe it's right to just take it out and add it back in later if we see fit.